feat: Add the possibility to set examples to the default handler behvior

bubu11e opened the pull request:
When trying to add examples for the default tonic status with the function fizz.Response like this:

```
routerGroup.POST(
            "", []fizz.OperationOption{
                fizz.Summary("My Summary"),
                fizz.ResponseWithExamples(
                    fmt.Sprintf("%d", http.StatusAccepted),
                    "toto", nil, nil, map[string]string{
                        "example ": "example",
                    }),
            },
            tonic.Handler(<the_handler>, http.StatusAccepted),
        )
```

we get the following error:

`panic: error while generating OpenAPI spec on operation XXXX : response with code XXX already exists [recovered,repanicked]`

This request add the possibility to set multiples examples for the default handler behavior.

wI2L commented on fizz.go:
```diff
@@ -130,48 +130,80 @@ func (g *RouterGroup) GinRouterGroup() *gin.RouterGroup {
 // Handle registers a new request handler that is wrapped
 // with Tonic and documented in the OpenAPI specification.
-func (g *RouterGroup) Handle(path, method string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
+func (g *RouterGroup) HandleWithExamples(path, method string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
```
Wrong comment, should be on `Handle`, and `HandleWithExamples` should have its own.

bubu11e committed:
feat: Add the possibility to set examples to the default handler behavior

When trying to add examples for the default tonic status with the function fizz.Response, we get the following error:

    panic: error while generating OpenAPI spec on operation XXXX : response with code XXX already exists [recovered, repanicked]

This commit allow to define examples for the default handler behavior

Signed-off-by: julien.girard <julien.girard@ovhcloud.com>

wI2L commented on fizz.go:
```diff
@@ -346,6 +346,13 @@ func ResponseWithExamples(statusCode, desc string, model interface{}, headers []
 	}
 }
 
+// ResponseWithExamples is a variant of Response that accept many examples.
```
Update comment to reflect the func name.

bubu11e replied:
Hi, done.

wI2L commented on openapi/generator.go:
```diff
@@ -1255,7 +1255,7 @@ func fieldNameFromTag(sf reflect.StructField, tagName string) string {
 }
 
-/// parseExampleValue is used to transform the string representation of the example value to the correct type.
+// / parseExampleValue is used to transform the string representation of the example value to the correct type.
```
Coukd you can remove the redundant `/` here ?

bubu11e replied:
Hi, done.
